sunxspex/photon_power_law.py

```diff
@@ -0,0 +1,204 @@
+import functools
+
+import numpy as np
+
+import astropy.units as u
+
+PHOTON_RATE_UNIT = u.ph / u.keV / u.cm**2 / u.s
+
+
+@u.quantity_input
+def power_law_integral(
+    energy: u.keV,
+    norm_energy: u.keV,
+    norm: PHOTON_RATE_UNIT,
+    index: u.one
+) -> (PHOTON_RATE_UNIT * u.keV):
+    r"""Evaluate the antiderivative of a power law at a given energy or vector of energies.
+
+    The power law antiderivative evaluated by this function is assumed to take the following form:
+    .. math:: f(E) = N \left( \frac{E}{E_0} \right)^{- \gamma},
```

Comment:
I think this needs be either like this for inline 

```
:math:`f(E) = N \left( \frac{E}{E_0} \right)^{- \gamma}` where
```

or like this for block mode the white space and newlines are necessary

```
.. math:: 
   f(E) = N \left( \frac{E}{E_0} \right)^{- \gamma}

where 
```

You can see the rendered docs for this PR here https://sunxspex--103.org.readthedocs.build/en/103/api/sunxspex.photon_power_law.broken_power_law_binned_flux.html#sunxspex.photon_power_law.broken_power_law_binned_flux